api/src/services/authorization.service.ts

```
@@ -62,8 +63,24 @@ export default class AuthorizationService {
+
+    const currentLocalesCount = membership.project.localesCount;
+    const currentTermsCount = membership.project.termsCount;
+
+    // SQLite-specific fix: SQLite updates the in-memory object after increment operations,
+    // causing double-counting in quota validation. Detect and correct this.
+    if (isDbType(DbType.BETTER_SQLITE3) && action === ProjectAction.ImportTranslation && newLocalesCount > 0 && newTermsCount > 0) {
+      // This is the second authorization call in import (with terms)
+      // If current locales count > 0, it means the locale was already added and counted
+      // Don't count it again
+      if (currentLocalesCount > 0) {
+        newLocalesCount = 0;
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)